content/java/concepts/calendar/terms/isSet/isSet.md

```diff
@@ -0,0 +1,63 @@
+---
+Title: '.isSet()'
+Description: 'Checks whether the given calendar field has a value set or not.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'Calendar'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`.isSet()`** method in `Calendar` class is used to check whether the given calendar field has a value set or not. This method returns `true` if the given calendar field has a value set, `false` otherwise. It returns true when the given parameter has been set by internal field interpretations. `isSet()` method is a non-static method, it is accessible with the class object.
```

Comment:
This definition is an exact match to other documentation, such as the link below. Whether this was intentional or not, this is plagiarism. There is no plagiarism permitted in docs. All entries must be original. Please revise the content to use your own words.
https://www.geeksforgeeks.org/calendar-isset-method-in-java-with-examples/#:~:text=The%20isSet(int%20calndr_field)%20method,by%20a%20get%20method%20call.